Modules/@babylonjs/react-native-windows/windows/CMakeLists.txt

```diff
@@ -5,8 +5,8 @@ project(ReactNativeBabylon)
 
 set(CMAKE_CXX_STANDARD 17)
 
-set(BABYLON_REACT_NATIVE_DIR "${CMAKE_CURRENT_LIST_DIR}/../../react-native")
-include(${BABYLON_REACT_NATIVE_DIR}/shared/CMakeLists.txt)
+set(BABYLON_REACT_NATIVE_DIR "${CMAKE_CURRENT_LIST_DIR}/../../react-native-iosandroid")
+include(${CMAKE_CURRENT_LIST_DIR}/../../react-native/shared/CMakeLists.txt)

```

Comment:
Why this change?

Reply:
Before the change, both included CMakeLists.txt and submodules were in the same `react-native` folder. But now react-native-iosandroid contains the submodules folder and the cmakelists.txt is in `react-native`. 2 folders instead of just 1 before.

Reply:
I see, I think the variable names are confusing now. Having `BABYLON_REACT_NATIVE_DIR` and `BABYLON_REACT_NATIVE_ANDROIDIOS` seems like it would be easier to understand.

Reply:
Fixed.